packages/desktop-client/vite.config.mts

```diff
@@ -152,7 +152,7 @@ export default defineConfig(async ({ mode }) => {
       mode === 'desktop'
         ? undefined
         : VitePWA({
-            registerType: 'autoUpdate',
+            registerType: 'prompt',
```

Comment:
'prompt' is misleading here. It's just needed so that we can hook into the "update ready" event. We only show a prompt if an update is available later